addons/website/static/src/snippets/s_hr/000.scss

```diff
@@ -7,5 +7,9 @@
         border-top: 1px solid currentColor;
         margin: 0;
         color: inherit;
+        // As BS5 added "opacity" and "background-color" on hr tag, we remove
+        // that here to let the user set the color as he want.
```

Comment:
@qsm-odoo Not 100% sure what comment you want here, let me know if I should change it